src/amo/components/HomeHero/index.js

```diff
@@ -0,0 +1,93 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import Link from 'amo/components/Link';
+import translate from 'core/i18n/translate';
+import Hero from 'ui/components/Hero';
+import type { I18nType } from 'core/types/i18n';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+type InternalProps = {|
+  i18n: I18nType,
+|};
+
+export class HomeHeroBase extends React.Component<InternalProps> {
+  getHeroes() {
+    const { i18n } = this.props;
+
+    // TODO: get urls
+    return [
+      {
+        title: i18n.gettext('Stay Safe Online'),
+        icon: 'stop-hand',
+        url: '/guides/stay-safe',
+      },
+      {
+        title: i18n.gettext('Organize Your Tabs'),
+        icon: 'browser',
+        url: '/guides/organize-your-tabs',
+      },
+      {
+        title: i18n.gettext('Enhance Your Media Experience'),
+        icon: 'video',
+        url: '/guides/elevate-your-media',
+      },
+    ];
+  }
+
+  getHeroHeader() {
+    const { i18n } = this.props;
+
+    return (
+      <div className="HomeHero-header">
+        <h2 className="HomeHero-title">
```

Comment:
That should probably be `HomeHero-header-title`, since it is inside `HomeHero-header`.